Library/Formula/shml.rb

```diff
@@ -0,0 +1,16 @@
+class Shml < Formula
+  desc "Style Framework for The Terminal"
+  homepage "http://shml.xyz"
+  url "https://github.com/MaxCDN/shml/archive/1.0.4.tar.gz"
+  sha256 "583fefbbf2adf45d989b052696be0f20f695903878558a4d2e9f74255b91ed51"
+
+  def install
+    bin.install "shml.sh"
```

Comment:
No, not really. I do want both executables available as typical use-case is to source this executable, and the more common pattern is `source shml.sh`. That said, I can either install both or install it as `shml` and then generate a symlink for `shml.sh`, if that's preferred.

I couldn't find any clear documentation on using `bin.install` and/or `bin.install_symlink`. I would assume that `bin.install "shml.sh" => "shml"` works?